libnavigation-base/src/main/java/com/mapbox/navigation/base/extensions/NavigationOptionsEx.kt

```diff
@@ -0,0 +1,20 @@
+@file:JvmName("NavigationOptionsExtensions")
+
+package com.mapbox.navigation.base.extensions
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.base.options.CopilotOptions
+import com.mapbox.navigation.base.options.NavigationOptions
+
+/**
+ * Defines configuration for Copilot.
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+fun NavigationOptions.Builder.copilotOptions(copilotOptions: CopilotOptions):
+    NavigationOptions.Builder = this.copilotOptions(copilotOptions)
+
+/**
+ * Copilot configuration.
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+fun NavigationOptions.copilotOptions(): CopilotOptions = this.copilotOptions
```

Comment:
Looks like we don't need these extensions, we can make corresponding functions and properties in `NavigationOptions` public and add annotations there.

Reply:
Thought about that and was wondering if that approach would have more impact from the client side if we have to change the API in the future, causing more affecting changes to developers. Let me take it for a quick spin and double check 👍

Reply:
Fixed and pushed. Ready for a final round of 👀